lib/src/component/error_boundary_api.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:meta/meta.dart';
```

Comment:
#nit missing copyright comment 

Also there's a few places where the copyright comment on new files has the wrong year (although maybe you were planning on doing that in a different PR - I can't remember)

Reply:
Yeah, I'm just gonna add/update all the copyrights in a separate PR. Thanks for checking!